tests/playwright/tests/Multisig.spec.ts

```diff
@@ -0,0 +1,123 @@
+import { test, expect } from '@playwright/test';
+
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/');
+  await page.getByRole('button', { name: 'Get started' }).click();
+  await page.getByRole('button', { name: 'Existing Wallet Import wallet' }).click();
+  await page.getByLabel('1:', { exact: true }).click();
+  await page.getByLabel('1:', { exact: true }).fill(process.env.TON_MNEMONIC_ANANAS);
+  await page.getByRole('button', { name: 'Continue' }).click();
+  await page.getByRole('button', { name: 'Continue' }).click();
+  await page.locator('#create-password').fill('a]HmC.;MAcQJ[+Y@&r!-3h');
+  await page.locator('#create-password-confirm').fill('a]HmC.;MAcQJ[+Y@&r!-3h');
+  await page.getByRole('button', { name: 'Continue' }).click();
+  await page.getByRole('button', { name: 'Save' }).click();
+});
+
+test.afterEach(async ({ page }) => {
+  await page.getByRole('link', { name: 'Settings' }).click();
+  await page.getByText('Delete Account').click();
+  await page
+    .locator('div')
+    .filter({ hasText: /^I have a backup copy of recovery phrase$/ })
+    .locator('div')
+    .click();
+  await page.getByRole('button', { name: 'Delete wallet data' }).click();
+});
+
+//Multisig creation (up to payment step)
+test('Create new multisig', async ({ page }) => {
+
+  //multisig page view
+  await page.getByRole('button', { name: 'Account 1 UQBA…OP8V W5 UQDH…' }).click();
+  await expect(page.getByRole('link', { name: 'Multisig Wallets' })).toBeVisible();
+  await page.getByRole('link', { name: 'Multisig Wallets' }).click();
+  await expect(page.getByText('Multisig WalletsNew Multisig')).toBeVisible();
+  await expect(page.getByText('Multisig UQBL…v_')).toBeVisible();
+  await page.getByText('👩‍💼Multisig UQBL…v_kAUQBL…').click();
+  await expect(page.getByText('👩‍💼Multisig UQBL…v_kAUQBL…')).toBeVisible();
+
+  await page.locator('.sc-kLKjoy > .sc-bXDltw').first().click();
+  await expect(page.locator('#root')).toContainText('👩‍💼Multisig UQBL…v_kAMultisig');
```

Comment:
Three a lot's of Multisigs, butter to clarify selection 

```
Expected string: "👩‍💼Multisig UQBL…v_kAMultisig" Received string: "Multisig UQBv…0IRaUQBv…0IRaMultisig👨‍🍳DashboardDiscover🍍Account 1UQBA…OP8VW5UQDH…TkZ5W5 betaUQD2…GzCiv4R2👨‍🍳Multisig UQBv…0IRaMultisigUQDj…pOuvv3R2UQBW…lnEFv3R1Add WalletPreferencesGet Pro$ 0.57SendReceiveBuyTokensHistoryCollectiblesDomainsSwapRequestsSettingsTokensTON0.09$ 5.79 −10.35%$ 0.57"
```

Reply:
I didn`t get it